.github/workflows/conda-python-build.yaml

```
@@ -51,44 +48,84 @@ permissions:
     statuses: none
 
 jobs:
-    build-test-upload:
-      strategy:
-        fail-fast: false
-      runs-on: linux-${{ inputs.CPU }}-gpu-${{ inputs.GPU }}-latest-1
+  compute-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      MATRIX: ${{ steps.compute-matrix.outputs.MATRIX }}
+    steps:
+      - name: Compute Build Matrix
+        id: compute-matrix
+        run: |
+          set -eo pipefail
+
+          # Please keep the matrices sorted in ascending order by the following:
+          #
+          #     [ARCH, PY_VER, CUDA_VER, LINUX_VER]
+          #
+          export MATRIX="
+          # amd64
+          - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '12.8.0', LINUX_VER: 'rockylinux8' }
+          - { ARCH: 'amd64', PY_VER: '3.11', CUDA_VER: '11.8.0', LINUX_VER: 'rockylinux8' }
```

This still says CTK 11.8, probably a typo. I'm going to suggest a change and commit here.
```suggestion
          - { ARCH: 'amd64', PY_VER: '3.11', CUDA_VER: '12.8.0', LINUX_VER: 'rockylinux8' }
```